Add plugin: Convert Ai Math

lights7 opened the pull request:
Try to follow template of pull request

# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/lights7/convert-ai-math

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

lights7 commented:
Finally!!!!

joethei commented:
[Copyright (c) 2024 Darko Pejakovic](https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/LICENSE#L3)
Update this, but remember to keep the original copyright

[Ai](https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/manifest.json#L3)
Both characters are capitalized here.

lights7 commented:
joethel,

Got you, thank you for your review. I just corrected Ai ti AI and updated.

lights7

> On Mar 14,2025, at 4:25 PM, Johannes Theiner ***@***.***> wrote:
> 
> 
> joethei
>  left a comment 
> (obsidianmd/obsidian-releases#5595)
> Copyright (c) 2024 Darko Pejakovic
> Update this, but remember to keep the original copyright
> 
> Ai
> Both characters are capitalized here.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
> 
>  <https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/LICENSE#L3>
>  <https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/manifest.json#L3> <https://github.com/obsidianmd/obsidian-releases/pull/5595#issuecomment-2725679007> <https://github.com/notifications/unsubscribe-auth/AALUH56XLMUKBHHNCLZSODL2UM3K7AVCNFSM6AAAAABYM2O42OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRVGY3TSMBQG4>
> 
> joethei
>  left a comment 
> (obsidianmd/obsidian-releases#5595)
>  <https://github.com/obsidianmd/obsidian-releases/pull/5595#issuecomment-2725679007>
> Copyright (c) 2024 Darko Pejakovic <https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/LICENSE#L3>
> Update this, but remember to keep the original copyright
> 
> Ai <https://github.com/lights7/convert-ai-math/blob/206ff154ea4a33bf0002431c64db4f00580ac257/manifest.json#L3>
> Both characters are capitalized here.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/obsidianmd/obsidian-releases/pull/5595#issuecomment-2725679007>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AALUH56XLMUKBHHNCLZSODL2UM3K7AVCNFSM6AAAAABYM2O42OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRVGY3TSMBQG4>.
> You are receiving this because you authored the thread.
>
